core/src/builtin_widgets.rs

```diff
@@ -187,6 +223,14 @@ impl<T> FatObj<T> {
     );
     self.host
   }
+
+  /// Return the LazyWidgetId of the host widget, through which you can access
+  /// the WidgetId after building.
+  pub fn lazy_host_id(&self) -> LazyWidgetId { self.builtin.lazy_host_id() }
+
+  /// Return the LazyWidgetId of the external widget (wrapped with the built-in
+  /// host), through which you can access the WidgetId after building.
```

Comment:
/// Return the LazyWidgetId point to `WidgetId` of the root of the sub widget tree after the
/// FatObj has built.